docs/tempo/website/api_docs/_index.md

```diff
@@ -115,6 +118,40 @@ Returns:
 By default this endpoint returns [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto/trace/v1) JSON,
 but if it can also send OpenTelemetry proto if `Accept: application/protobuf` is passed.
 
+### Search
```

Comment:
I think we should mention here that this is gated behind a feature flag and also we should mark all of this experimental/subject to change. I don't think we want to give 1.x stability guarantees on these endpoints.

Reply:
Add similar color and phrasing as Loki experimental apis: https://grafana.com/docs/loki/latest/api/#list-rule-groups